_migration-assistant/migration-phases/reroute-source-to-proxy.md

```diff
@@ -63,29 +63,50 @@ Note the records in the logging topic.
    
 After a short period, re-execute the same command again and compare the increased number of records against the expected HTTP requests.
 
-## Backfilling documents to the source cluster
+## Troubleshooting
 
-From your source cluster snapshot, you can begin backfilling documents into the target cluster. Once you have started this process, a fleet of workers will spin up to read the snapshot and reindex documents into the target cluster. This fleet of workers can be scaled to increase the speed at which documents are reindexed into the target cluster.
+The following sections may be helpful in diagnosing common issues that arise. If you encounter an issue not documented, incorporate that feedback into this section for future readers.
 
-### Checking the starting state of the clusters
+### Host header routing configuration
 
-You can check the indexes and document counts of the source and target clusters by running the `cat-indices` command. This can be used to monitor the difference between the source and target for any migration scenario. Check the indexes of both clusters using the following command:
+Some systems, such as Elastic Cloud and other hosted Elasticsearch services, use the Host header for routing traffic to the appropriate cluster. When using the Capture Proxy with these systems, you need to configure the proxy to set the Host header to match your source cluster's domain.
```

Comment:
I would add to the end of this sentence something like "your source cluster's domain since clients may be setting the host otherwise to the proxy".  Make it clear that this isn't a proxy bug but a more general client/configuration issue.